packages/react-core/src/components/LoginPage/examples/LoginPageLanguageSelect.tsx

```diff
@@ -0,0 +1,184 @@
+          <path d="M448 56.7v398.5c0 13.7-11.1 24.7-24.7 24.7H309.1V306.5h58.2l8.7-67.6h-67v-43.2c0-19.6 5.4-32.9 33.5-32.9h35.8v-60.5c-6.2-.8-27.4-2.7-52.2-2.7-51.6 0-87 31.5-87 89.4v49.9h-58.4v67.6h58.4V480H24.7C11.1 480 0 468.9 0 455.3V56.7C0 43.1 11.1 32 24.7 32h398.5c13.7 0 24.8 11.1 24.8 24.7z" />
+        </svg>
+      </LoginMainFooterLinksItem>
+      <LoginMainFooterLinksItem href="#" linkComponentProps={{ 'aria-label': 'Login with Gitlab' }}>
+        <svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+          <path d="M29.782 199.732L256 493.714 8.074 309.699c-6.856-5.142-9.712-13.996-7.141-21.993l28.849-87.974zm75.405-174.806c-3.142-8.854-15.709-8.854-18.851 0L29.782 199.732h131.961L105.187 24.926zm56.556 174.806L256 493.714l94.257-293.982H161.743zm349.324 87.974l-28.849-87.974L256 493.714l247.926-184.015c6.855-5.142 9.711-13.996 7.141-21.993zm-85.404-262.78c-3.142-8.854-15.709-8.854-18.851 0l-56.555 174.806h131.961L425.663 24.926z" />
+        </svg>
+      </LoginMainFooterLinksItem>
+    </React.Fragment>
+  );
+
+  const signUpForAccountMessage = (
+    <LoginMainFooterBandItem>
+      Need an account? <a href="#">Sign up.</a>
+    </LoginMainFooterBandItem>
+  );
+
+  const forgotCredentials = (
+    <LoginMainFooterBandItem>
+      <a href="#">Forgot username or password?</a>
+    </LoginMainFooterBandItem>
+  );
+
+  const listItem = (
+    <React.Fragment>
+      <ListItem>
+        <LoginFooterItem href="#">Terms of Use </LoginFooterItem>
+      </ListItem>
+      <ListItem>
+        <LoginFooterItem href="#">Help</LoginFooterItem>
+      </ListItem>
+      <ListItem>
+        <LoginFooterItem href="#">Privacy Policy</LoginFooterItem>
+      </ListItem>
+    </React.Fragment>
+  );
+
+  const loginForm = (
+    <LoginForm
+      showHelperText={showHelperText}
+      helperText="Invalid login credentials."
+      helperTextIcon={<ExclamationCircleIcon />}
+      usernameLabel="Username"
+      usernameValue={username}
+      onChangeUsername={handleUsernameChange}
+      isValidUsername={isValidUsername}
+      passwordLabel="Password"
+      passwordValue={password}
+      onChangePassword={handlePasswordChange}
+      isValidPassword={isValidPassword}
+      rememberMeLabel="Keep me logged in for 30 days."
+      isRememberMeChecked={isRememberMeChecked}
+      onChangeRememberMe={onRememberMeClick}
+      onLoginButtonClick={onLoginButtonClick}
+      loginButtonLabel="Log in"
+    />
+  );
+
+  const images = {
+    lg: '/assets/images/pfbg_1200.jpg',
+    sm: '/assets/images/pfbg_768.jpg',
+    sm2x: '/assets/images/pfbg_768@2x.jpg',
+    xs: '/assets/images/pfbg_576.jpg',
+    xs2x: '/assets/images/pfbg_576@2x.jpg'
+  };
+
+  return (
+    <LoginPage
+      footerListVariants={ListVariant.inline}
+      brandImgSrc={brandImg2}
+      brandImgAlt="PatternFly logo"
+      backgroundImgSrc={images}
+      backgroundImgAlt="Images"
```

Comment:
Since this is a background image, I'd say we could omit this prop so that it defaults as an empty string. Typically for images that are more decorative like this, passing in `alt=""` is a better option since assistive technologies won't really announce the existence of that image, but for users who rely on sight it'll still be visible (assuming no internet speed issues or anything).

Reply:
Ah thanks for pointing that out! I forget that more than the selected line may appear. In this case I'm referring to `backgroundImgAlt`. Usually a site logo like in this case can benefit from alt text to help convey what site a user is on, so we can keep `brandImgAlt` as it is.